### PR TITLE
release-1.18 branch: Updated config.toml for release v1.19

### PR DESCRIPTION
**Pull request by savitharaghunathan**
Updated config.toml for release v1.19

/hold

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -115,13 +115,13 @@ time_format_blog = "Monday, January 02, 2006"
...
+latest = "v1.19"
 
-fullversion = "v1.18.0"
+fullversion = "v1.18.8"
 version = "v1.18"
-githubbranch = "master"
-docsbranch = "master"
-deprecated = false
+githubbranch = "v1.18.8"
+docsbranch = "release-1.18"
```
Comment (sftim) [inline config.toml:123]: This doesn't look right to me. I think this should be:
```suggestion
fullversion = "v1.19.0"
version = "v1.19"
githubbranch = "master"
docsbranch = "master"
deprecated = false
```

**Review by sftim — COMMENTED** [review]
@savitharaghunathan I think these changes would mark v1.19 as deprecated (and that that isn't what you intend). Please check against the suggestion I've made.

**Review by sftim — COMMENTED** [review]
D'oh! I'm all wrong here. My last review was wrong. Sorry!

**Review by sftim — COMMENTED** [review]
/lgtm

**Comment by jimangel** [comment]
/lgtm
/approve
Per discussion [here](https://kubernetes.slack.com/archives/C1J0BPD2M/p1598466973088100?thread_ts=1598447979.070600&cid=C1J0BPD2M), I am manually merging this PR as part of the 1.19 release. 🎉

**Pull request merged**